studeecloud/client
# Refactoring video component code

This PR:
- Attempts to fix the video rendering issue by setting the videos to panel to hidden during work mode. The implementation works but it does not solve the rendering issue
- Deletes the failed attempt to remove event listeners
- Refactored some video code out of `App.js` and into a directory for the top-right panel

To test this PR:
Run the client locally and make sure it works as before. The video rendering bug will still occur but other behaviour should be unaffected.
